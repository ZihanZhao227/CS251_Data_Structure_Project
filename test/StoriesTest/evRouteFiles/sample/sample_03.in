5
0 0 0 CITY StartCity
1 20 10 CHARGING_STATION ChargingStation1
2 30 0 RESTAURANT Restaurant1
3 40 20 CHARGING_STATION ChargingStation2
4 50 0 CITY DestinationCity