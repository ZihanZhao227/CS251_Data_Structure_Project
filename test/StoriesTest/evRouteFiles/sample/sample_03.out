0 4 60.0 50.000
0 4